### PR TITLE
apollo-compiler@0.7.1

### PR DESCRIPTION
**Pull request by lrlna**
spotted a bug with #501 primarily, which needed a patch, but let's just merge everything else that's new anyways

### CODE REVIEW

File: crates/apollo-compiler/CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +18,26 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
 
-# [0.7.0](https://crates.io/crates/apollo-compiler/0.7.0) - 2023-03-27
+# [0.7.1](https://crates.io/crates/apollo-compiler/0.7.1) - 2023-03-28
+
+## Features
+- validate indirectly self-referential directives by [goto_bus_stop] in [pull/494]
+
+## Fixes
+- include built-in enum types in db.type_system() query by [Simon Sapin] in [pull/501]
+- field.field_definition() works for interface types by [zackangelo] in [pull/502]
```
Comment (goto-bus-stop) [inline crates/apollo-compiler/CHANGELOG.md:28]:
```suggestion
- `field.field_definition()` works for interface types by [zackangelo] in [pull/502]
```

File: crates/apollo-compiler/CHANGELOG.md [hunk 2]
```diff
@@ -18,7 +18,26 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
 ## Documentation -->
 
-# [0.7.0](https://crates.io/crates/apollo-compiler/0.7.0) - 2023-03-27
+# [0.7.1](https://crates.io/crates/apollo-compiler/0.7.1) - 2023-03-28
+
+## Features
+- validate indirectly self-referential directives by [goto_bus_stop] in [pull/494]
+
+## Fixes
+- include built-in enum types in db.type_system() query by [Simon Sapin] in [pull/501]
```
Comment (goto-bus-stop) [inline crates/apollo-compiler/CHANGELOG.md:27]:
```suggestion
- include built-in enum types in `db.type_system()` query by [SimonSapin] in [pull/501]
```

File: crates/apollo-compiler/CHANGELOG.md [hunk 3]
```diff
@@ -18,7 +18,26 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Documentation -->
 
-# [0.7.0](https://crates.io/crates/apollo-compiler/0.7.0) - 2023-03-27
+# [0.7.1](https://crates.io/crates/apollo-compiler/0.7.1) - 2023-03-28
+
+## Features
+- validate indirectly self-referential directives by [goto_bus_stop] in [pull/494]
```
Comment (goto-bus-stop) [inline crates/apollo-compiler/CHANGELOG.md:24]:
```suggestion
- validate indirectly self-referential directives by [goto-bus-stop] in [pull/494]
```

**Review by goto-bus-stop — APPROVED** [review]

File: crates/apollo-compiler/CHANGELOG.md (hunk 3 above)
Comment (lrlna) [reply crates/apollo-compiler/CHANGELOG.md:24]: why do i keep doing this!! ahhh

**Pull request merged**